frontend/package.json

```diff
@@ -61,6 +64,8 @@
     "lint:fix": "npm run lint -- --fix",
     "postinstall": "nuxt prepare",
     "preview": "nuxt preview",
+    "test:e2e": "npx playwright test --config=tests/playwright.config.ts",
+    "test:debug": "npx playwright test --config=tests/playwright.config.ts --ui",
```

Comment:
I like `ui` better as it's is more `generic`, because one might want to use `--ui` mode without doing `debugging` but just to see what's going on in with `headed browser`